web_application/Backend/FlaskApp/app.py

```diff
@@ -697,6 +697,15 @@ def calculate_cofficients():
     #except DoesNotExist:
     #    return jsonify({'message': 'Collaboration not found'}), 404
 
```

Comment:
```suggestion
def get_metadata(metadata_ids: List[str]):
    raise NotImplementedError()
```
Create template function like this. Do similar things to other functions.

Reply:
Remember to declare types for inputs